halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/ServiceSettings.java

```diff
@@ -48,6 +50,7 @@
   Map<String, String> env = new HashMap<>();
   String artifactId;
   boolean enabled;
+  boolean auth;
```

Comment:
Agreed on the verb change.

It's per-service basic-auth, I should instead move this to the `SpringServiceSettings` subclass since that's what it's actually modifying. That's orthogonal to SSL